google/cloud/bigtable/examples/bigtable_hello_world.cc

```diff
@@ -33,32 +33,32 @@ int main(int argc, char* argv[]) try {
   std::string const table_id = argv[3];
 
   // Connect to the Cloud Bigtable Admin API.
-  //! [connect admin] [START connecting_to_bigtable]
+  //! [connect admin] [START bigtable_hw_connect]
   google::cloud::bigtable::TableAdmin table_admin(
       google::cloud::bigtable::CreateDefaultAdminClient(
           project_id, google::cloud::bigtable::ClientOptions()),
       instance_id);
-  //! [connect admin] [END connecting_to_bigtable]
+  //! [connect admin] [END bigtable_hw_connect]
 
-  //! [create table] [START creating_a_table]
+  //! [create table] [START bigtable_hw_create_table]
   // Define the desired schema for the Table.
   auto gc_rule = google::cloud::bigtable::GcRule::MaxNumVersions(1);
   google::cloud::bigtable::TableConfig schema({{"family", gc_rule}}, {});
 
   // Create a table.
   auto returned_schema = table_admin.CreateTable(table_id, schema);
-  //! [create table] [END creating_a_table]
+  //! [create table]
 
   // Create an object to access the Cloud Bigtable Data API.
-  //! [connect data] [START connecting_to_bigtable]
+  //! [connect data]
```

Comment:
"START bigtable_hw_create_table" ?

Reply:
This is the same as above, I am effectively merging the `connecting_to_bigtable` and `creating_a_table` region tags into a single `bigtable_hw_create_table` region.

Reply:
Ack.